agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -495,11 +506,40 @@ async def get_user_object(user_uid: str) -> UserDB:
 
             return create_user
         else:
-            raise Exception("Please login or signup")
+            return None
     else:
         return user
 
 
+async def get_users_by_ids(user_ids: List) -> List:
+    """
+    Retrieve users from the database by their IDs.
+
+    Args:
+        user_ids (List): A list of user IDs to retrieve.
+
+    Returns:
+        List: A list of dictionaries representing the retrieved users.
+    """
+
+    users_db: List[UserDB] = await engine.find(UserDB, UserDB.id.in_(user_ids))
+
+    return users_db
+
+
+async def save_object_to_db(object_to_save: Any) -> Any:
```

Comment:
what is this? why do we need it?

Reply:
@mmabrouk 
To keep all db operations and avoid calling engine any where outside db_manager in ee, I defined this function to replace the usage of `await engine.save(object)`

This was done to address the issue you mention [here](https://github.com/Agenta-AI/agenta_ee/pull/110#discussion_r1358432987)

See an example use [here](https://github.com/Agenta-AI/agenta_ee/blob/ed63c4af65c692b4ad2eac9964ff9fd49cd3c171/ee/agenta-backend/agenta_backend/ee/services/organization_service.py#L58) and see the application in this [PR](https://github.com/Agenta-AI/agenta_ee/pull/110)

Reply:
Thanks for the explanation. Not a big fan of the solution, it's basically renaming engine.save by save_object_to_db. Nevertheless, I don't have a better solution right now.